### DISCUSSION TITLE
Wrong transaction fee

### DISCUSSION THREAD

**Discussion by VladicaKostic** [Chia-Network/chia-blockchain#9218] [Support]
Hi,
I made a transaction that is confirmed at height 1085039 where i accidently put high transaction fee. Is there a way so i can get those xch back?
Best regards,
Vladica Kostic

**Comment by emlowe** [comment]
Unfortunately no. Some lucky farmer thanks you

**Comment by tibetburak** [comment]
merhaba. 
Şu yükseklikte onaylandı 1108245 , onaylanan fakat işlem ücretini yanlış girdiğim için xch askıda kaldı. yardımcı olabilir misiniz ?
